libs/agno/agno/agent/agent.py

```diff
@@ -3981,7 +4036,7 @@ async def aprint_response(
                 if render:
                     live_log.update(Group(*panels))
 
-                _arun_generator = await self.arun(
+                async for resp in await self.arun(
```

Comment:
Are you sure about this one? I would also think so, but just check

Reply:
seems to work fine, let me cross check